src/App.tsx

```diff
@@ -17,18 +68,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                filterByStatus={filterByStatus}
+                filterByQuery={filterByQuery}
+                query={query}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length
+                ? (
+                  <TodoList
+                    todos={filteredTodos}
+                    selectedTodo={selectedTodo}
+                    setterOfSelectedTodo={setterOfSelectedTodo}
```

Comment:
Check naming convention - https://javascript.plainenglish.io/handy-naming-conventions-for-event-handler-functions-props-in-react-fc1cbb791364 
```suggestion
                    onTodoSelection={setterOfSelectedTodo}
```